### PR TITLE
Updates content for About page

### PR DESCRIPTION
**Pull request by MelSumner**
### :pushpin: Summary

If merged, this PR updates the content for the About page to use paragraph content instead of cards.

:speech_balloon: Please consider using [conventional comments](https://conventionalcomments.org/) when reviewing this PR.

### CODE REVIEW

**Comment by MelSumner** [comment]
@hashicorp/hds-design I need some help on the "getting started for designers" section.

File: website/app/templates/about.hbs [hunk 1]
```diff
@@ -2,7 +2,24 @@
...
-    <Doc::Cards::Deck @cols="4" @cards={{this.cards.about}} />
+  <Doc::Page::Content>
+    <p class="doc-text-body">The HashiCorp Design System is a flexible, scalable design system built on the foundations of accessibility, beautiful design, and consistent user experience.</p>
+    <h2 class="doc-text-h2">Overview</h2>
+    <p class="doc-text-body">The Design Systems Team creates and maintains the HashiCorp Design System. This includes the Figma components, the web components, the documentation, and more. Read through our <a href="/about/overview/" target="_blank" rel="noopener noreferrer">overview</a> for more details, and to see why adopting the HashiCorp Design System is the future for product teams.</p>
+    
+    <h2 class="doc-text-h2">Principles</h2>
+    <p class="doc-text-body">It is important that we stay focused and grounded as a team. Setting clear expectations for both ourselves and our users by defining our philosophy and values through our <a href="/about/principles/" target="_blank" rel="noopener noreferrer">principles</a> help us do just that.</p> 
+
+    <h2 class="doc-text-h2">Accessibility Statement</h2>
```
Comment (heatherlarsen) [inline website/app/templates/about.hbs:13]:
```suggestion
    <h2 class="doc-text-h2">Accessibility statement</h2>
```

File: website/app/templates/about.hbs [hunk 2]
```diff
@@ -2,7 +2,24 @@
...
+    <p class="doc-text-body">Accessibility is not just a requirement, it is a part of our company culture, grounded in <a href="https://www.hashicorp.com/our-principles" rel="noopener noreferrer" target="_blank">our company principles</a> such as integrity, communication and kindness. Read more about our intentions for an accessible design system in our <a href="/about/accessibility-statement/" target="_blank" rel="noopener noreferrer">accessibility statement</a>.</p>
+    
+    <h2 class="doc-text-h2">Getting Started for Designers</h2>
```
Comment (heatherlarsen) [inline website/app/templates/about.hbs:16]:
```suggestion
    <h2 class="doc-text-h2">Getting started for designers</h2>
```

File: website/app/templates/about.hbs [hunk 3]
```diff
@@ -2,7 +2,24 @@
...
+    <p class="doc-text-body">Learn how to use our components and other resources by reading our guide to <a href="/getting-started/for-designers/" target="_blank" rel="noopener noreferrer">getting started for designers</a>.</p>
+    
+    <h2 class="doc-text-h2">Getting Started for Engineers</h2>
```
Comment (heatherlarsen) [inline website/app/templates/about.hbs:19]:
```suggestion
    <h2 class="doc-text-h2">Getting started for engineers</h2>
```

File: website/app/templates/about.hbs [hunk 4]
```diff
@@ -2,7 +2,24 @@
...
+    <p class="doc-text-body">While the design system components are conveniently packaged in a standard Ember addon, we still recommend reading through our guide to <a href="/getting-started/for-engineers/" target="_blank" rel="noopener noreferrer">getting started for engineers</a> to better understand some specific conventions we have put in place to help engineers focus on what is really important in their application, instead of thinking about component APIs.</p>
+    
+    <h2 class="doc-text-h2">Contributions</h2>
+    <p class="doc-text-body">As an open-source project, we think a lot about how people can contribute-- and there are quite a few ways that our users are encouraged to participate! Read our <a href="/getting-started/contribution/" target="_blank" rel="noopener noreferrer">contribution</a> documentation to find the different ways anyone can contribute to the HashiCorp Design System.</p>
```
Comment (heatherlarsen) [inline website/app/templates/about.hbs:23]:
```suggestion
    <p class="doc-text-body">As an open-source project, we think a lot about how people can contribute-- and there are quite a few ways our consumers are encouraged to participate! Read our <a href="/getting-started/contribution/" target="_blank" rel="noopener noreferrer">contribution</a> documentation to find the different ways anyone can contribute to Helios.</p>
```

**Review by heatherlarsen — APPROVED** [review]
I really love this ❤️ 🎉 Concise and flows very well! I left a few nit-picky suggestions and name changes.

File: website/app/templates/about.hbs [hunk 5]
```diff
@@ -2,7 +2,24 @@
 
 <Doc::Page::Stage>
   <Doc::Page::Cover @title="About" />
-  <Doc::Page::Content @breakthrough={{true}}>
-    <Doc::Cards::Deck @cols="4" @cards={{this.cards.about}} />
+  <Doc::Page::Content>
+    <p class="doc-text-body">The HashiCorp Design System is a flexible, scalable design system built on the foundations of accessibility, beautiful design, and consistent user experience.</p>
+    <h2 class="doc-text-h2">Overview</h2>
+    <p class="doc-text-body">The Design Systems Team creates and maintains the HashiCorp Design System. This includes the Figma components, the web components, the documentation, and more. Read through our <a href="/about/overview/" target="_blank" rel="noopener noreferrer">overview</a> for more details, and to see why adopting the HashiCorp Design System is the future for product teams.</p>
```
[the remark this answers is not in the document]
Comment (MelSumner) [reply website/app/templates/about.hbs:8]: Wait, we're changing our team name too?

[the remark this answers is not in the document]
Comment (Dhaulagiri) [reply website/app/templates/about.hbs:8]: We had discussed this briefly in a slack thread, but for now let's leave the team name as-is and only update the reference to the design system

File: website/app/templates/about.hbs [hunk 6]
```diff
@@ -2,7 +2,24 @@
 
 <Doc::Page::Stage>
   <Doc::Page::Cover @title="About" />
-  <Doc::Page::Content @breakthrough={{true}}>
-    <Doc::Cards::Deck @cols="4" @cards={{this.cards.about}} />
+  <Doc::Page::Content>
+    <p class="doc-text-body">Helios is a flexible, scalable design system built on the foundations of accessibility, beautiful design, and consistent user experience.</p>
+    <h2 class="doc-text-h2">Overview</h2>
+    <p class="doc-text-body">The Design Systems Team creates and maintains the Helios Design System. This includes the Figma components, the web components, the documentation, and more. Read through our <a href="/about/overview/" target="_blank" rel="noopener noreferrer">overview</a> for more details, and to see why adopting the HashiCorp Design System is the future for product teams.</p>
+
```
Comment (KristinLBradley) [inline website/app/templates/about.hbs:9]: Nit: I would remove the extra "the's" to be less repetitive.

"This includes the Figma components, the web components, the documentation, and more."
=>
"This includes the Figma components, web components, documentation, and more."

File: website/app/templates/about.hbs [hunk 7]
```diff
@@ -2,7 +2,24 @@
...
+    
+    <h2 class="doc-text-h2">Principles</h2>
+    <p class="doc-text-body">It's important that we stay focused and grounded as a team. Setting clear expectations for both ourselves and our consumers by defining our philosophy and values through our <a href="/about/principles/" target="_blank" rel="noopener noreferrer">principles</a> help us do just that.</p> 
+
+    <h2 class="doc-text-h2">Accessibility statement</h2>
+    <p class="doc-text-body">Accessibility is not just a requirement, it is a part of our company culture, grounded in <a href="https://www.hashicorp.com/our-principles" rel="noopener noreferrer" target="_blank">our company principles</a> such as integrity, communication and kindness. Read more about our intentions for an accessible design system in our <a href="/about/accessibility-statement/" target="_blank" rel="noopener noreferrer">accessibility statement</a>.</p>
+
```
Comment (KristinLBradley) [inline website/app/templates/about.hbs:15]: Missing Oxford comma:

"integrity, communication and kindness"
=>
"integrity, communication, and kindness"

File: website/app/templates/about.hbs [hunk 8]
```diff
@@ -2,7 +2,24 @@
...
+    <p class="doc-text-body">Learn how to use our components and other resources by reading our guide to <a href="/getting-started/for-designers/" target="_blank" rel="noopener noreferrer">getting started for designers</a>.</p>
+    
+    <h2 class="doc-text-h2">Getting started for engineers</h2>
+    <p class="doc-text-body">While the design system components are conveniently packaged in a standard Ember addon, we still recommend reading through our guide to <a href="/getting-started/for-engineers/" target="_blank" rel="noopener noreferrer">getting started for engineers</a> to better understand some specific conventions we have put in place to help engineers focus on what is really important in their application, instead of thinking about component APIs.</p>
+    
+    <h2 class="doc-text-h2">Contributions</h2>
+    <p class="doc-text-body">As an open-source project, we think a lot about how people can contribute-- and there are quite a few ways our consumers are encouraged to participate! Read our <a href="/getting-started/contribution/" target="_blank" rel="noopener noreferrer">contribution</a> documentation to find the different ways anyone can contribute to Helios.</p>
```
Comment (KristinLBradley) [inline website/app/templates/about.hbs:23]: Use em dash, & remove space:

"... how people can contribute-- and ..."
=>
"... how people can contribute—and ..."

**Review by KristinLBradley — COMMENTED** [review]
Looks good! Just left a few minor comments.

File: website/app/templates/about.hbs [hunk 9]
```diff
@@ -2,7 +2,24 @@
 
 <Doc::Page::Stage>
   <Doc::Page::Cover @title="About" />
-  <Doc::Page::Content @breakthrough={{true}}>
-    <Doc::Cards::Deck @cols="4" @cards={{this.cards.about}} />
+  <Doc::Page::Content>
+    <p class="doc-text-body">Helios is a flexible, scalable design system built on the foundations of accessibility, beautiful design, and consistent user experience.</p>
+    <h2 class="doc-text-h2">Overview</h2>
+    <p class="doc-text-body">The Design Systems Team creates and maintains the Helios Design System. This includes the Figma components, the web components, the documentation, and more. Read through our <a href="/about/overview/" target="_blank" rel="noopener noreferrer">overview</a> for more details, and to see why adopting the HashiCorp Design System is the future for product teams.</p>
```
Comment (MelSumner) [inline website/app/templates/about.hbs:8]:
```suggestion
    <p class="doc-text-body">The Design Systems Team creates and maintains the Helios Design System. This includes the Figma components, web components, documentation, and more. Read through our <a href="/about/overview/" target="_blank" rel="noopener noreferrer">overview</a> for more details, and to see why adopting the HashiCorp Design System is the future for product teams.</p>
```

File: website/app/templates/about.hbs [hunk 10]
```diff
@@ -2,7 +2,24 @@
...
+    
+    <h2 class="doc-text-h2">Principles</h2>
+    <p class="doc-text-body">It's important that we stay focused and grounded as a team. Setting clear expectations for both ourselves and our consumers by defining our philosophy and values through our <a href="/about/principles/" target="_blank" rel="noopener noreferrer">principles</a> help us do just that.</p> 
+
+    <h2 class="doc-text-h2">Accessibility statement</h2>
+    <p class="doc-text-body">Accessibility is not just a requirement, it is a part of our company culture, grounded in <a href="https://www.hashicorp.com/our-principles" rel="noopener noreferrer" target="_blank">our company principles</a> such as integrity, communication and kindness. Read more about our intentions for an accessible design system in our <a href="/about/accessibility-statement/" target="_blank" rel="noopener noreferrer">accessibility statement</a>.</p>
```
Comment (MelSumner) [inline website/app/templates/about.hbs:14]:
```suggestion
    <p class="doc-text-body">Accessibility is not just a requirement, it is a part of our company culture, grounded in <a href="https://www.hashicorp.com/our-principles" rel="noopener noreferrer" target="_blank">our company principles</a> such as integrity, communication, and kindness. Read more about our intentions for an accessible design system in our <a href="/about/accessibility-statement/" target="_blank" rel="noopener noreferrer">accessibility statement</a>.</p>
```

File: website/app/templates/about.hbs [hunk 11]
```diff
@@ -2,7 +2,24 @@
...
   <Doc::Page::Cover @title="About" />
-  <Doc::Page::Content @breakthrough={{true}}>
-    <Doc::Cards::Deck @cols="4" @cards={{this.cards.about}} />
+  <Doc::Page::Content>
+    <p class="doc-text-body">Helios is a flexible, scalable design system built on the foundations of accessibility, beautiful design, and consistent user experience.</p>
+    <h2 class="doc-text-h2">Overview</h2>
+    <p class="doc-text-body">The Design Systems Team creates and maintains the Helios Design System. This includes the Figma components, the web components, the documentation, and more. Read through our <a href="/about/overview/" target="_blank" rel="noopener noreferrer">overview</a> for more details, and to see why adopting the HashiCorp Design System is the future for product teams.</p>
+    
+    <h2 class="doc-text-h2">Principles</h2>
+    <p class="doc-text-body">It's important that we stay focused and grounded as a team. Setting clear expectations for both ourselves and our consumers by defining our philosophy and values through our <a href="/about/principles/" target="_blank" rel="noopener noreferrer">principles</a> help us do just that.</p>
```
Comment (MelSumner) [inline website/app/templates/about.hbs:11]:
```suggestion
    <p class="doc-text-body">It’s important that we stay focused and grounded as a team. Setting clear expectations for both ourselves and our consumers by defining our philosophy and values through our <a href="/about/principles/" target="_blank" rel="noopener noreferrer">principles</a> help us do just that.</p> 
```

File: website/app/templates/about.hbs (hunk 8 above)
Comment (MelSumner) [inline website/app/templates/about.hbs:23]:
```suggestion
    <p class="doc-text-body">As an open-source project, we think a lot about how people can contribute—and there are quite a few ways our consumers are encouraged to participate! Read our <a href="/getting-started/contribution/" target="_blank" rel="noopener noreferrer">contribution</a> documentation to find the different ways anyone can contribute to Helios.</p>
```

**Review by KristinLBradley — APPROVED** [review]

**Pull request merged**